nuxeo-features/nuxeo-platform-imaging/nuxeo-platform-imaging-jsf/src/main/java/org/nuxeo/ecm/platform/picture/web/PictureBookManagerBean.java

```diff
@@ -243,21 +243,19 @@ protected void addBlobHolderToZip(String path, ZipOutputStream out, byte[] data,
         }
         for (Blob content : blobs) {
             String fileName = content.getFilename();
-            if (content != null) {
-                // Workaround to deal with duplicate file names.
-                int tryCount = 0;
-                while (true) {
-                    try {
-                        if (tryCount == 0) {
-                            ZipUtils._zip(path + fileName, content.getStream(), out);
-                        } else {
-                            ZipUtils._zip(path + formatFileName(fileName, "(" + tryCount + ")"), content.getStream(),
-                                    out);
-                        }
-                        break;
-                    } catch (ZipException e) {
-                        tryCount++;
+            // Workaround to deal with duplicate file names.
+            int tryCount = 0;
+            while (true) {
```

Comment:
Is there no max trycount?  ie, if ` ZipUtils._zip` continues to fail does this loop continue for ever?

Reply:
It seems so...